examples/oneapi/cpp/source/jaccard/jaccard_batch_app.cpp

```diff
@@ -39,22 +39,22 @@ using namespace std;
 /// @param [in]   block_column_count The size of block by columns
 template <class Graph>
 void vertex_similarity_block_processing(const Graph &g,
-                                        const int32_t block_row_count,
-                                        const int32_t block_column_count);
+                                        int32_t block_row_count,
+                                        int32_t block_column_count);
 
 int main(int argc, char **argv) {
   // load the graph
-  std::string filename = get_data_path("graph.csv");
-  graph_csv_data_source ds(filename);
-  load_graph::descriptor<> d;
-  auto graph = load_graph::load(d, ds);
+  const std::string filename = get_data_path("graph.csv");
```

Comment:
line 32: `using namespace std` - need to be removed?

Reply:
"using namespace std" was used as there are a lot of number variables, like, std::int32_t and using namespace std reduces amount of code. This problem can be resolved in another way, if we put a using especially on std::int32_t with purpose not to extend each init of int32_t with std:: prefix. But such approach was rejected by the team as it complicates the example. 

Suggest to remove std:: from each functionality as using do it.

Reply:
> line 32: `using namespace std` - need to be removed?

"using namespace std" was used as there are a lot of number variables, like, std::int32_t and using namespace std reduces amount of code. This problem can be resolved in another way, if we put a using especially on std::int32_t with purpose not to extend each init of int32_t with std:: prefix. But such approach was rejected by the team as it complicates the example.

Suggest not to write using, and extend all variables with std::

Reply:
Why `using std::int64_t` complicates the code?